### ISSUE TITLE
[BUG]: Starter project is broken

### ISSUE THREAD

**Issue by markmals** [analogjs/analog#207]
### Please provide the environment you discovered this bug in.

Local development and [StackBlitz starter project](https://analogjs.org/new)

### Which area/package is the issue in?

create-analog

### Description

When you create a new Analog project using `npm init analog` locally or using the StackBlitz starter and start the dev server, `[plugin angular-compiler]` throws several errors.

### Please provide the exception or error you saw

```true
✘ [ERROR] [plugin angular-compiler] Do not know how to load path: angular:styles/component:;2;/Users/mmalstrom/Developer/test/analog-project/src/app/routes/index.ts

✘ [ERROR] [plugin angular-compiler] Do not know how to load path: angular:styles/component:;1;/Users/mmalstrom/Developer/test/analog-project/src/app/app.component.ts
```

### I would be willing to submit a PR to fix this issue

- [ ] Yes
- [X] No

**Comment by brandonroberts** [comment]
Thanks @markmals. Must be something with Angular 15.1

**Comment by brandonroberts** [comment]
Fix has been included in the `0.2.0-alpha.22` release of `@analogjs/vite-plugin-angular`
